PWGGA/GammaConv/AliCaloPhotonCuts.cxx

```diff
@@ -4367,10 +4367,11 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
             energy /= FunctionNL_kSDM(energy, 0.972156, -4.10515, -0.381273);
             energy /= FunctionNL_kSDM(energy, 0.979999, -4.39136, -0.102332);
           }
-          if(fClusterType==3) energy /= FunctionNL_kSDM(energy, 0.980211, -4.374598, -0.171988);
+          if(fClusterType==3 && fCurrentMC==k16k5a) energy /= 0.9870110951;
+          if(fClusterType==3 && fCurrentMC==k17e2) energy /= FunctionNL_kSDM(energy0.986513, 0.430032, -10.99999);
```

Comment:
please try to do a local compilation before pushing - this will crash for sure

Reply:
I'm really sorry for this typo, I'll fix it and compile it

Reply:
i pushed an fix